src/pages/index.js

```diff
@@ -100,36 +146,95 @@ function fillProfileForm() {
     descInput.value = info.description;
 }
 
-function updateInfo(event, data) {
-    userInfo.setUserInfo({
+async function updateInfo(event, data) {
+    const newInfo = {
         name: data["name-input"],
         description: data["description-input"],
+    };
+    userInfo.setUserInfo(newInfo);
```

Comment:
You need to update `DOM` only in blocks `then` after a successful response from the server so that when there is an error it wouldn’t change


**Please, correct it everywhere**